includes/admin/class-sensei-learner-management.php

```diff
@@ -136,17 +135,18 @@ public function add_custom_navigation() {
 	 */
 	private function display_students_navigation( WP_Screen $screen ) {
 		?>
-		<div id="sensei-custom-navigation" class="sensei-custom-navigation">
-			<div class="sensei-custom-navigation__heading-with-info">
-				<div class="sensei-custom-navigation__title">
-					<h1><?php esc_html_e( 'Students', 'sensei-lms' ); ?></h1>
+			<div id="sensei-custom-navigation" class="sensei-custom-navigation">
+				<div class="sensei-custom-navigation__heading-with-info">
+					<div class="sensei-custom-navigation__title">
+						<h1><?php esc_html_e( 'Students', 'sensei-lms' ); ?></h1>
+					</div>
+					<div class="sensei-custom-navigation__separator"></div>
+					<a class="sensei-custom-navigation__info" target="_blank" href="https://senseilms.com/documentation/student-management?utm_source=plugin_sensei&utm_medium=docs&utm_campaign=student-management">
+						<?php echo esc_html__( 'Guide To Student Management', 'sensei-lms' ); ?>
+					</a>
 				</div>
-				<div class="sensei-custom-navigation__separator"></div>
-				<a class="sensei-custom-navigation__info" target="_blank" href="https://senseilms.com/documentation/student-management?utm_source=plugin_sensei&utm_medium=docs&utm_campaign=student-management">
-					<?php echo esc_html__( 'Guide To Student Management', 'sensei-lms' ); ?>
-				</a>
+				<hr class="wp-header-end">
```

Comment:
What do you think of moving the `.wp-header-end` element away from the navigation and directly to the students inner page? My idea is to move it right after where the navigation would end up after it's moved by the JS code. I think in this case the element could be moved in these two files:
* [html-admin-page-students-main.php](https://github.com/Automattic/sensei/blob/0b955afa5fa81ea7d0fb1a4e5a0c66bff1f10ee3/includes/admin/views/html-admin-page-students-main.php#L26) (inside the `.wrap` element, before the action)
* [html-admin-page-students-course.php](https://github.com/Automattic/sensei/blob/0b955afa5fa81ea7d0fb1a4e5a0c66bff1f10ee3/includes/admin/views/html-admin-page-students-course.php#L23) (before the `#poststuff` element)